docs/src/tutorial.md

```diff
@@ -65,15 +70,27 @@ function jump_example(n::Int)
 end
 
 jumpnlp_example = jump_example(n)
-pqn_jumpnlp = PartiallySeparableNLPModel(jumpnlp_example)
+pqn_jumpnlp = PSNLPModel(jumpnlp_example)
 
 fx = NLPModels.obj(pqn_jumpnlp, x) # compute the obective function
 gx = NLPModels.grad(pqn_jumpnlp, x) # compute the gradient
 ```
 In version v0.2.0, [`ManualNLPModel`](https://github.com/JuliaSmoothOptimizers/ManualNLPModels.jl)s will be supported.
 
-## A partitioned quasi-Newton `NLPModel`
-When defining a `PartiallySeparableNLPModel`, you allocate storage for partitioned quasi-Newton updates, which are implemented in `PartitionedStructures.jl`.
+## Partitioned quasi-Newton `NLPModel`s
+A model deriving from `AbstractPQNNLPModel<:AbstractPartiallySeparableNLPModel` allocates storage required for partitioned quasi-Newton updates, which are implemented in `PartitionedStructures.jl` (see the [PartitionedStructures.jl tutorial](https://juliasmoothoptimizers.github.io/PartitionedStructures.jl/dev/tutorial/) for more details).
+There are several variants:
+* 'PBFGSNLPModel': every element-Hessian approximation is updated with BFGS;
+* 'PSR1NLPModel': every element-Hessian approximation is updated with SR1;
+* 'PSENLPModel': every element-Hessian approximation is updated with BFGS if the curvature condition holds, or withSR1 otherwise;
+* 'PCSNLPModel': each element-Hessian approximation with BFGS if it is classified as `convex`, or with SR1 otherwise;
+* 'PLBFGSNLP': every element-Hessian approximations is a LBFGS operator;
+* 'PLSR1NLPModel': every element-Hessian approximations is a LSR1 operator;
+* 'PLSENLPModel': by default, every element-Hessian approximations is a LBFGS operator as long as the curvature condition holds, otherwise it becomes a LSR1 operator.
```

Comment:
Put model types between backticks instead of quotes.

Reply:
There is no PLCSNLPModel. Is that intentional? (I don't mean to add one to a list that is already long; I'm just wondering.)

Reply:
It is not intentionnal.
I implemented it for dense element matrices, I did not extended it to linear-operator element matrices.
I created an [issue](https://github.com/JuliaSmoothOptimizers/PartitionedStructures.jl/issues/66) in PartitionedStructures.jl for this addition.